compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1828,7 +1828,7 @@ rustc_queries! {
         desc { "normalizing `{}`", goal.value }
     }
 
-    query implied_outlives_bounds(
+    query implied_outlives_bounds_compat(
```

Comment:
Given that it seems like most places still use this, it seems like it would make sense to keep this name the same, and name the new version v2 or something.